src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -50,6 +53,9 @@ contract QuestionAPI is Ownable, NFTLocked {
         _questionStateController.initializeQuestion(questionId);
         _claimController.initializeQuestion(questionId, claimLimit);
 
+        // Lock Metric for the question
+        _vault.lockMetric(questionId, _msg.sender(), 1e18);
```

Comment:
either way you approach the comment above - the cost here (1e18) should come from the costController